src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,140 @@
+package duke;
+import exceptions.DukeDeadlineException;
+import exceptions.DukeException;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Deals with loading tasks from the file and saving tasks in the file
+ */
+public class Storage {
+    private File file;
+
+    /**
+     * Constructor
+     * Create a storage file in given filepath
```

Comment:
```suggestion
     * Creates a storage file in given filepath
```
The first sentence should start with creates instead of create